presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/rest-openapi.yml

```diff
@@ -0,0 +1,170 @@
+        - in: path
+          name: functionName
+          schema:
+            $ref: '#/components/schemas/QualifiedObjectName/properties/objectName'
+          required: true
+          description: The return type of the function of the user to get
+        - in: query
+          name: returnType
+          schema:
+            $ref: '#/components/schemas/TypeSignature/properties/base'
+          required: true
+          description: The return type of the function
+        - in: query
+          name: numArgs
+          schema:
+            type: integer
+          required: true
+          description: The numbers of arguments passed in
+        - in: query
+          name: argType
+          schema:
+            type: object
+            # $ref: '#/components/schemas/TypeSignature/parameters'
+            additionalProperties:
+              type: string
+              pattern: '^[A-Za-z][A-Za-z0-9]*$'
+            example:
+              argType1: string
+              argType2: double
+          description: The type for each argument passed in
+      requestBody:
+        required: true
+        content:
+          text/plain; charset=utf-8:
+            schema:
+              $ref: '#/components/schemas/SerializedPage'
+              type: object
+              properties:
+                username:
+                  type: string
+                  format: binary
+      responses:
+        '200':
+          description: function value output
+          content:
+            text/plain; charset=utf-8:
+              schema:
+                $ref: '#/components/schemas/SerializedPageBinary'
+components:
+  schemas:
+    QualifiedObjectName:
+      type: object
+      properties:
+        catalogName:
+          type: string
+          description: The name of the catalog
+        schemaName:
+          type: string
+          description: The name of the Schema
+        objectName:
+          type: string
+          description: The name of the function
+    TypeSignature:
+      type: object
+      properties:
+        base:
+          type: object
+          description: The base signature type
+        parameters:
+          type: array
+          items:
+            $ref: '#/components/schemas/TypeSignatureParameter'
+          description: list of type signature parameters
+        calculated:
+          type: boolean
+        IDENTIFIER_PATTERN:
+          type: object
+          description: The pattern of identification
+        BASE_NAME_ALIAS_TO_CANONICAL:
+          type: object
+        SIMPLE_TYPE_WITH_SPACES:
+          type: object
+        BIGINT_ENUM_PREFIX:
+          type: string
+        ENUM_PREFIX:
+          type: object
+        DISTINCT_TYPE_PREFIX:
+          type: object
+    TypeSignatureParameter:
+      type: object
+      properties:
+        kind:
+          type: object
+          description: The parameter kind
+        value:
+          type: object
+          description: The value
+    Page:
```

Comment:
Likewise, this won't work.  Just make it a binary payload and link to the Presto docs.